compiler/src/main/resources/dxCompiler_runtime.conf

```diff
@@ -0,0 +1,8 @@
+dxCompiler {
+  regionToProject = [
+    {
+      region = "aws:us-east-1"
+      path = "dxCompiler_playground:/builds/jdidion/2.0.0-rc6"
```

Comment:
Was this added accidentally?

Reply:
yes thanks for catching. I've updated .gitignore.